.changelog/3967.changed.txt

```diff
@@ -0,0 +1 @@
+release preparation for v4.15.1
```

Comment:
there isn't a new changelog required for the release PR. In this PR we use `make update-changelog` to consolidate change logs to the markdown file (CHANGELOG.md)

Reply:
Use the skip-changelog label for this PR